### PR TITLE
[CLOUDSTACK-10301] Allow updating the network ACL list name and Description

### PR DESCRIPTION
**Pull request by rafaelweingartner**
Currently, it is not possible to update the Network ACL name or Description of an ACL after it has been created. It might be interesting to expose this option via API and to add this feature to the UI.

The Screenshot shows the fields that this PR will allow updating.
editacllistdata

### CODE REVIEW

**Comment by rafaelweingartner** [comment]
@blueorangutan package

File: engine/schema/src/main/java/com/cloud/network/vpc/NetworkACLVO.java [hunk 1]
```diff
@@ -101,4 +101,12 @@ public void setVpcId(long vpcId) {
     public boolean isDisplay() {
         return display;
     }
+
```
[the remark this answers is not in the document]
Comment (DaanHoogland) [reply engine/schema/src/main/java/com/cloud/network/vpc/NetworkACLVO.java]: never mind, dumb remark

File: server/src/test/java/com/cloud/network/vpc/NetworkACLServiceImplTest.java [hunk 2]
```diff
@@ -64,9 +67,11 @@
     @Mock
     private NetworkACLItemDao networkAclItemDaoMock;
     @Mock
-    private EntityManager EntityManagerMock;
+    private EntityManager entityManagerMock;
     @Mock
-    private AccountManager AccountManager;
+    private AccountManager AccountManagerMock;
```
Comment (DaanHoogland) [inline server/src/test/java/com/cloud/network/vpc/NetworkACLServiceImplTest.java]: lower case for this var name as well?

Comment (rafaelweingartner) [reply server/src/test/java/com/cloud/network/vpc/NetworkACLServiceImplTest.java]: Sure, it was a case of "ctrl+v" fail

File: ui/l10n/nl_NL.js [hunk 3]
```diff
@@ -313,6 +313,7 @@ var dictionary = {
     "label.add.accounts": "Voeg accounts toe",
     "label.add.accounts.to": "Voeg accounts toe aan",
     "label.add.acl.list": "voeg een ACL lijst toe",
+    "label.edit.acl.list": "Edit ACL List",
```
[the remark this answers is not in the document]
Comment (rafaelweingartner) [reply ui/l10n/nl_NL.js]: Thanks ;)

**Comment by rafaelweingartner** [comment]
@DaanHoogland I applied your suggestions, but I did not understand what you said about "annotations API" and audit trail. Could you explain that further?

**Comment by DaanHoogland** [comment]
@rafaelweingartner user comments can be added to a list (enum) of entity types:         HOST("host"), DOMAIN("domain"), VM("vm_instance")
at this moment with the annotations API. addAnnotation, etc. This is extendable to other entities. The annotations left on entities this way leaves an audit trail by user id. Useful for admins.

**Comment by rafaelweingartner** [comment]
@DaanHoogland, this PR is not for a comment/description in ACL. This is to allow editing ACL listing description and name. Once you create an ACL listing, right now, you are not allowed to edit the name and description. This is quite annoying, if you have a typo or some incomplete sentence you cannot change them. You would need to remove and add the ACL again (or you can go directly in the database like I have done already so many times).

**Comment by rafaelweingartner** [comment]
@blueorangutan package

**Comment by rafaelweingartner** [comment]
@borisstoyanov can you run tests here?
This is a simple, but useful feature in production.

**Comment by borisstoyanov** [comment]
sure @rafaelweingartner 
@blueorangutan test

**Comment by rafaelweingartner** [comment]
@borisstoyanov do you have the test results here?

**Comment by borisstoyanov** [comment]
Deployment has failed @rafaelweingartner, let me restart 
@blueorangutan test

**Comment by borisstoyanov** [comment]
@blueorangutan package

**Comment by borisstoyanov** [comment]
@blueorangutan test

**Comment by rafaelweingartner** [comment]
@borisstoyanov these errors do not seem to be caused by changes introduced here.

**Pull request merged**